schema/src/ProjectInfo.yaml

```diff
@@ -191,6 +187,13 @@ classes:
   Any:
     class_uri: linkml:Any
 
+slots:
+  projectStartDate:
+    slot_uri: schema:startDate
+    description: The start date of the project.
+  projectEndDate:
+    slot_uri: schema:endDate
+    description: The end date of the project.
```

Comment:
Also need to add these to the ProjectInfo `slot_usage` if moving them to slots.

But I did just test this change, and moving these to slots fixes the weird validation error/message we were getting @clevinson ! I wonder if it is because `CreditProjectInfo` class uses `slot_usage` to make the start + end date `required`, but it was not technically a slot.